src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -513,6 +571,7 @@ public void setJob(final AbstractJob job)
      * @return the job this citizen has.
      */
     @Nullable
+    @SuppressWarnings("squid:S1166")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1166' warning is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)